wiktor_hardware_interface/JointValueQuantity measured_joint_position
wiktor_hardware_interface/JointValueQuantity commanded_joint_position
wiktor_hardware_interface/JointValueQuantity measured_joint_velocity
wiktor_hardware_interface/JointValueQuantity measured_joint_torque
wiktor_hardware_interface/JointValueQuantity estimated_external_torque
wiktor_hardware_interface/CartesianValueQuantity measured_cartesian_pose
wiktor_hardware_interface/CartesianValueQuantity commanded_cartesian_pose
wiktor_hardware_interface/CartesianValueQuantity estimated_external_wrench
std_msgs/Header header
uint8 IS_POSITION_MOTION=0
uint8 IS_CARTESIAN_MOTION=1
uint8 IS_IMPEDANCE_CONTROL=2
uint8 JOINT_POSITION=0
uint8 JOINT_IMPEDANCE=2
uint8 CARTESIAN_POSE=1
uint8 CARTESIAN_IMPEDANCE=3
uint8 active_control_mode
